packages/react-native-win32/overrides.json

```diff
@@ -222,6 +222,10 @@
       "type": "platform",
       "file": "src/Libraries/Components/View/Tests/ViewWin32Test.tsx"
     },
+    {
+      "type": "platform",
```

Comment:
Haven't associated these or given them a bug since the plan is to remove these overrides directly after backporting.